{"name": "risk user -- @1.14 cases", "status": "passed", "steps": [{"name": "Given risk makes get request: /reconciliation/user_data/", "status": "passed", "start": 1611572613412, "stop": 1611572613451}, {"name": "Then check response of request: You do not have permission to perform this action. is False", "status": "passed", "start": 1611572613451, "stop": 1611572613451}], "parameters": [{"name": "url", "value": "/reconciliation/user_data/"}, {"name": "method", "value": "get"}, {"name": "bool", "value": "False"}], "start": 1611572613411, "stop": 1611572613451, "uuid": "d7e50b9b-cb99-4780-bf6f-d0aa339f8805", "historyId": "4361e83895bf2d6b085d01c690dedd16", "fullName": "features/II-Reconciliation/III-reconciliation_access.feature:risk user", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "reconciliation check access"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}